dds_web/api/user.py

```diff
@@ -924,13 +967,14 @@ def post(self):
                 ),
             ) from err
 
-        flask.current_app.logger.debug(
-            f"User {existing_user.username} no longer associated with project {project.public_id}."
-        )
+        if unanswered_invite:
+            msg = f"Invited user is no longer associated with {project.public_id}."
+        else:
+            msg = f"User with email {user_email} no longer associated with {project.public_id}."
```

Comment:
Should just be able to define these in the if statements above instead. (see suggestions further up in the code)
```suggestion
```